### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by DelhinRharl**
##In this Pull Request:

- [ ] I have added the required files for the project, including images, css, html and javascript
- [ ] I edited my readme.md file to suit the project.
- [ ] I recorded a video explaining the project
- [ ] Link to the video (https://www.loom.com/share/fb87e34585a64c1d8271df37a96ceaa3)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,67 @@
...
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- HTML
+- CSS
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```
Comment (rotimiazeez) [inline README.md]: - [ ] Kindly add the deployed link of your page to the Live demo link for easy access to your page😊

File: about.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      </div>
+      <span class="divide"></span>
+      <div class="part2">
+        <div class="rights"><p>All Rights Reserved.</p></div>
+        <div class="web"><p>www.cbfest@walker.com</p></div>
+        <div class="contact">
+          <p>Tel - +47 -234-5000-327/+42-254-5010-300</p>
+        </div>
+      </div>
+    </footer>
```
Comment (rotimiazeez) [inline about.html]: - [x] Please note that the footer should have a white background on the mobile screen and a dark background on the desktop screen to match the given requirement.👌

File: about.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  </head>
+  <body>
+    <header class="header">
+      <nav class="my-nav">
+        <div class="top-bar">
+          <a href=""
+            ><img class="insta" src="./images/instagram-30.png" alt=""
+          /></a>
+          <p>Cindy Shin</p>
+        </div>
```
Comment (rotimiazeez) [inline about.html]: - [x] Please note that the top short bar section should contain social media links, logout, and language links aligned to the right as seen [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)✔

File: about.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      </div>
+    </section>
+
+    <section class="past-festivals">
+      <h3>As it happened in the previous years:</h3>
+      <img src="./images/festa.jpg" alt="2020 festival image" />
+      <p>2020:Nairobi</p>
+      <img src="./images/festb.jpg" alt="2019 festival image" />
+      <p>2019: Belgium</p>
+    </section>
```
Comment (rotimiazeez) [inline about.html]: - [x] Please make sure that images of this section are organized in a horizontal row as stated as seen in the [original design.](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)😎

File: about.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+            </li>
+            <li>
+              <a href="#programme" id="programpage"><p>Program</p></a>
+            </li>
+            <li>
+              <a href="#sponserr" id="sponsorpage"><p>Sponsor</p></a>
+            </li>
+            <li>
+              <a href="" id="newspage"><p>News</p></a>
+            </li>
```
Comment (rotimiazeez) [inline about.html]: - [x] Please note that on the` About` desktop page there is no link to navigate to the home page Kindly fix that😉

File: index.html [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+        <li class="card6">
+          <img id="img6" src="." alt="Alan Walker" />
+          <div class="cname">
+            <h4 id="hf"></h4>
+            <h5 id="h6"></h5>
+            <p id="p6"></p>
+          </div>
+        </li>
+      </ul>
+    </section>
```
Comment (rotimiazeez) [inline index.html]: - [x] Please make sure this section is created dynamically without hard-coding and only loads when the page loads👀
- for example, you can have:  ```  <section class="headliners" id="headliners">
      <div class="artistss" id="artistss"></div>``` and the content are created dynamically like👇
 ```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
#Hi @DelhinRharl 👋, 

### Good job so far!🎉
### Summary👏
- No linter error✔
- Descriptive Readme👌
- Good commit messages✔
#### There are some issues that you still need to work on to go to the next project but you are almost there!
> Kindy make the suggested changes and you'll be unstoppable🚀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 2 above)
Comment (DelhinRharl) [reply about.html]: ## Noted 👍🏽 

- [ ] only the about page should have this, since the main page has white background

File: about.html (hunk 5 above)
Comment (DelhinRharl) [reply about.html]: Fixed!! 👍🏽

File: index.html (hunk 6 above)
Comment (DelhinRharl) [reply index.html]: Fixed!

File: about.html (hunk 4 above)
Comment (DelhinRharl) [reply about.html]: Fixed...

File: about.html (hunk 3 above)
Comment (DelhinRharl) [reply about.html]: fixed..

**Review by rotimiazeez — APPROVED** [review]
# Hi @DelhinRharl 👏, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀💯
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**